src/pages/Host.tsx

```diff
@@ -30,6 +37,14 @@ export const Host: React.FC = () => {
         hasData
         size="large"
       />
+      <PageLayout>
+        <PageSider width={350}>
+          <Metadata loading={loading} data={data} error={error} />
+          <Code language="shell" multiline={false}>
```

Comment:
I think setting multiline to true would be better for this so we get that overflow which i think is fine since the user has a copy button now.
![image](https://user-images.githubusercontent.com/4605522/89553531-a25c7780-d7db-11ea-9604-fcb1fc60a1a7.png)

Reply:
good point! thank you!